composer.json

```diff
@@ -13,27 +13,31 @@
       "name": "Evan Lovely"
     }
   ],
-  "repositories": [
-    {
+  "repositories": {
+    "boltvcs": {
       "type": "vcs",
-      "url": "https://github.com/boltdesignsystem/twig-tools",
+      "url": "https://github.com/sreenivasulu-paruchuri/twig-tools.git",
```

Comment:
Do you not have access to the bolt design system twig-tools repo? Or is this just temporary for testing?